cpp/daal/include/sycl/internal/buffer_utils.h

```diff
@@ -129,10 +129,16 @@ class BufferConverterTo
     {
         _st = services::Status();
 
-        auto buffer    = _src.template get<DataType>();
-        auto subbuffer = buffer.getSubBuffer(_offset, _size);
-
-        _dest = subbuffer;
+        auto buffer = _src.template get<DataType>();
```

Comment:
Can you please also create a specialization of `operator()` in `BufferConverterFrom` when it is no need to perform conversion?